docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
 New and Improved
 ----------------
 
-- Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See :ref:`Superplot documentation <WorkbenchSuperplot>` for more information.
+- **Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See** :ref:`Superplot documentation <WorkbenchSuperplot>` **for more information.**
 
 .. figure:: ../../images/superplot_1.png
     :width: 500px
     :align: center
 
-- Workflow diagrams in help pages are now ``.svg`` rather than ``.png``
+- **New widget and workbench plugin:** :ref:`WorkbenchWorkspaceCalculator` **, allows users to perform binary operations and scaling by a floating number on workspaces;
```

Comment:
This sentence doesn't read well, could you rephrase it?

Reply:
I've had a look at the documentation for this which is similar to this release note but perhaps gives a little more detail - 'The workspace calculator allows to quickly perform binary operations on selected workspace pairs along with simultaneous scaling by arbitrary floating point numbers, provided they are different from zero.'

Maybe I should use this in the release notes instead?

Reply:
Perhaps this is more concise:
`The workspace calculator allows to quickly perform binary operations on selected workspace pairs, optionally with scaling.`